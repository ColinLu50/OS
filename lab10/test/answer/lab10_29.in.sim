34152
1658
1176
931
1176
927
