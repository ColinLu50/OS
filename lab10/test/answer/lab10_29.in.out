FCFS
236 -> 532 -> 436 -> 669 -> 102 -> 554 -> 201 -> 179 -> 549 -> 376 -> 523 -> 644 -> 229 -> 430 -> 886 -> 89 -> 320 -> 344 -> 841 -> 338 -> 790 -> 286 -> 509 -> 787 -> 413 -> 583 -> 329 -> 630 -> 55 -> 519 -> 488 -> 412 -> 423 -> 894 -> 429 -> 865 -> 933 -> 205 -> 158 -> 0 -> 884 -> 691 -> 33 -> 852 -> 63 -> 798 -> 852 -> 608 -> 67 -> 108 -> 370 -> 940 -> 152 -> 853 -> 109 -> 182 -> 302 -> 939 -> 917 -> 704 -> 112 -> 7 -> 511 -> 484 -> 901 -> 539 -> 907 -> 536 -> 601 -> 308 -> 11 -> 868 -> 492 -> 913 -> 440 -> 734 -> 620 -> 665 -> 787 -> 68 -> 436 -> 320 -> 391 -> 294 -> 825 -> 912 -> 926 -> 430 -> 249 -> 657 -> 40 -> 328 -> 520 -> 69 -> 701 -> 104 -> 34 -> 174 -> 392 -> 753 -> 486
34152
SSTF
236 -> 229 -> 249 -> 286 -> 294 -> 302 -> 308 -> 320 -> 320 -> 328 -> 329 -> 338 -> 344 -> 370 -> 376 -> 391 -> 392 -> 412 -> 413 -> 423 -> 429 -> 430 -> 430 -> 436 -> 436 -> 440 -> 484 -> 486 -> 488 -> 492 -> 509 -> 511 -> 519 -> 520 -> 523 -> 532 -> 536 -> 539 -> 549 -> 554 -> 583 -> 601 -> 608 -> 620 -> 630 -> 644 -> 657 -> 665 -> 669 -> 691 -> 701 -> 704 -> 734 -> 753 -> 787 -> 787 -> 790 -> 798 -> 825 -> 841 -> 852 -> 852 -> 853 -> 865 -> 868 -> 884 -> 886 -> 894 -> 901 -> 907 -> 912 -> 913 -> 917 -> 926 -> 933 -> 939 -> 940 -> 205 -> 201 -> 182 -> 179 -> 174 -> 158 -> 152 -> 112 -> 109 -> 108 -> 104 -> 102 -> 89 -> 69 -> 68 -> 67 -> 63 -> 55 -> 40 -> 34 -> 33 -> 11 -> 7 -> 0
1658
SCAN
236 -> 229 -> 205 -> 201 -> 182 -> 179 -> 174 -> 158 -> 152 -> 112 -> 109 -> 108 -> 104 -> 102 -> 89 -> 69 -> 68 -> 67 -> 63 -> 55 -> 40 -> 34 -> 33 -> 11 -> 7 -> 0 -> 0 -> 249 -> 286 -> 294 -> 302 -> 308 -> 320 -> 328 -> 329 -> 338 -> 344 -> 370 -> 376 -> 391 -> 392 -> 412 -> 413 -> 423 -> 429 -> 430 -> 436 -> 440 -> 484 -> 486 -> 488 -> 492 -> 509 -> 511 -> 519 -> 520 -> 523 -> 532 -> 536 -> 539 -> 549 -> 554 -> 583 -> 601 -> 608 -> 620 -> 630 -> 644 -> 657 -> 665 -> 669 -> 691 -> 701 -> 704 -> 734 -> 753 -> 787 -> 790 -> 798 -> 825 -> 841 -> 852 -> 853 -> 865 -> 868 -> 884 -> 886 -> 894 -> 901 -> 907 -> 912 -> 913 -> 917 -> 926 -> 933 -> 939 -> 940
1176
C-SCAN
236 -> 229 -> 205 -> 201 -> 182 -> 179 -> 174 -> 158 -> 152 -> 112 -> 109 -> 108 -> 104 -> 102 -> 89 -> 69 -> 68 -> 67 -> 63 -> 55 -> 40 -> 34 -> 33 -> 11 -> 7 -> 0 -> 0 -> 944 -> 940 -> 939 -> 933 -> 926 -> 917 -> 913 -> 912 -> 907 -> 901 -> 894 -> 886 -> 884 -> 868 -> 865 -> 853 -> 852 -> 841 -> 825 -> 798 -> 790 -> 787 -> 753 -> 734 -> 704 -> 701 -> 691 -> 669 -> 665 -> 657 -> 644 -> 630 -> 620 -> 608 -> 601 -> 583 -> 554 -> 549 -> 539 -> 536 -> 532 -> 523 -> 520 -> 519 -> 511 -> 509 -> 492 -> 488 -> 486 -> 484 -> 440 -> 436 -> 430 -> 429 -> 423 -> 413 -> 412 -> 392 -> 391 -> 376 -> 370 -> 344 -> 338 -> 329 -> 328 -> 320 -> 308 -> 302 -> 294 -> 286 -> 249
931
LOOK
236 -> 229 -> 205 -> 201 -> 182 -> 179 -> 174 -> 158 -> 152 -> 112 -> 109 -> 108 -> 104 -> 102 -> 89 -> 69 -> 68 -> 67 -> 63 -> 55 -> 40 -> 34 -> 33 -> 11 -> 7 -> 0 -> 249 -> 286 -> 294 -> 302 -> 308 -> 320 -> 328 -> 329 -> 338 -> 344 -> 370 -> 376 -> 391 -> 392 -> 412 -> 413 -> 423 -> 429 -> 430 -> 436 -> 440 -> 484 -> 486 -> 488 -> 492 -> 509 -> 511 -> 519 -> 520 -> 523 -> 532 -> 536 -> 539 -> 549 -> 554 -> 583 -> 601 -> 608 -> 620 -> 630 -> 644 -> 657 -> 665 -> 669 -> 691 -> 701 -> 704 -> 734 -> 753 -> 787 -> 790 -> 798 -> 825 -> 841 -> 852 -> 853 -> 865 -> 868 -> 884 -> 886 -> 894 -> 901 -> 907 -> 912 -> 913 -> 917 -> 926 -> 933 -> 939 -> 940
1176
C-LOOK
236 -> 229 -> 205 -> 201 -> 182 -> 179 -> 174 -> 158 -> 152 -> 112 -> 109 -> 108 -> 104 -> 102 -> 89 -> 69 -> 68 -> 67 -> 63 -> 55 -> 40 -> 34 -> 33 -> 11 -> 7 -> 0 -> 940 -> 939 -> 933 -> 926 -> 917 -> 913 -> 912 -> 907 -> 901 -> 894 -> 886 -> 884 -> 868 -> 865 -> 853 -> 852 -> 841 -> 825 -> 798 -> 790 -> 787 -> 753 -> 734 -> 704 -> 701 -> 691 -> 669 -> 665 -> 657 -> 644 -> 630 -> 620 -> 608 -> 601 -> 583 -> 554 -> 549 -> 539 -> 536 -> 532 -> 523 -> 520 -> 519 -> 511 -> 509 -> 492 -> 488 -> 486 -> 484 -> 440 -> 436 -> 430 -> 429 -> 423 -> 413 -> 412 -> 392 -> 391 -> 376 -> 370 -> 344 -> 338 -> 329 -> 328 -> 320 -> 308 -> 302 -> 294 -> 286 -> 249
927
